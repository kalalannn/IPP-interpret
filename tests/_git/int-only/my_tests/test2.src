.ippcode19

defvar GF@some

read GF@some int
write GF@some
break

