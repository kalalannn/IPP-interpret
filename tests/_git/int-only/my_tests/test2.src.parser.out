<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
    <instruction order="1" opcode="DEFVAR">
        <arg1 type="var">GF@some</arg1>
    </instruction>
    <instruction order="2" opcode="READ">
        <arg1 type="var">GF@some</arg1>
        <arg2 type="type">int</arg2>
    </instruction>
    <instruction order="3" opcode="WRITE">
        <arg1 type="var">GF@some</arg1>
    </instruction>
    <instruction order="4" opcode="BREAK"/>
</program>
